scripts/custom_lint_checks.py

```diff
@@ -347,6 +377,12 @@ def visit_raise(self, node):
         self._add_raise_message(missing_excs, func_node)
 
     def visit_return(self, node):
+        """Called for return. Checks and adds message regarding return.
```

Comment:
"Visits a function node that contains a return statement and verifies that the return value and the return type are documented."

Reply:
done.